pkg/gameservers/migration.go

```diff
@@ -211,3 +210,17 @@ func (mc *MigrationController) syncGameServer(ctx context.Context, key string) e
 
 	return nil
 }
+
+func (mc *MigrationController) anyAddressMatch(node *k8sv1.Node, gs *agonesv1.GameServer) bool {
```

Comment:
Nice! I like it.

One addition I'd like to see is a unit test for `anyAddressMatch` so we can check a few different variations of input.